src/Client/app/(admin)/exams/review/page.tsx

```
@@ -446,75 +661,151 @@
+                                                                    aiScores[
+                                                                        submission
+                                                                            .questionId
+                                                                    ] ??
+                                                                    submission.score ??
+                                                                    0
+                                                                ).toString()}
+                                                                onChange={(e) =>
+                                                                    updateWrittenSubmission(
+                                                                        submission.questionId,
+                                                                        {
+                                                                            score: parseInt(
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)